### DISCUSSION TITLE
21.08.25.수 회의록

### DISCUSSION THREAD

**Discussion by damilog** [Team-IT-DA/Frontend#98] [회의록]
- [x] 장바구니 내 상품별 배송비 정보 추가 필요 (데이지 -> BE)
```
 {
        "productId": 202,
        "productName": "유기농 감자 1박스",
        "price": 20000,
        "imgUrl": "https://www.naver.com",
        "productCount": 6
      }
```
<img width="316" alt="스크린샷 2021-08-25 오후 4 03 39" src="https://user-images.githubusercontent.com/56783350/130742293-60bf03d7-81d2-4c54-8315-f745df6865dd.png">

- 장바구니 조회에서 상품별 배송비 데이터도 주세요!
- 프론트에서 장바구니 수량 + 하고 나서 확인 버튼 누를 때 요청 보내는 방식으로 하기
- 프론트에서는 장바구니 수량 수정할 때 장바구니에 있는 모든 데이터를 보내줘야함
- 장바구니 수정 -> 받은 데이터 형식 그대로 보내줘야함
- 백엔드 : 상세페이지에서 장바구니 담기에 쓰일 API 추가 구현 예정

### Jenny
- [x] addProduct 최적화(form 상태 관리 방법 여러가지 시도해보기)
- [x] addProduct Header 분리
- [x] my page 상품 후기 쪽 view 

### Dico
- [x] sideDrawer에 stepper button 추가 
- [ ] side drawer 수량 수정 후 클릭하는 확인 버튼 추가 (데이지가 atom으로 만들예정)
- [ ] 장바구니 삭제 구현 
- [ ] my page tab color 바꾸기 

### Daisy

- [x] 장바구니 수량 증가에 따른 결제 상세 변경 로직 
- [x] 제품 삭제로직
- [x] atom 확인버튼 만들기
- [x] Header 리팩토링
- [ ] 배송지 입력 form view(시간되면)

**Comment by ha3158987** [comment]
MyPage에 적용할 tab을 Material UI에서 가져왔는데 children을 어떻게 받아와서 쓰는지 파악하려고 고전중인 상태입니다...ㅠㅠ
문법 때문인건지 뭔가 하나에서 너무 헤매는 게 아닌가 싶어서 회의때 같이 코드를 봐주셨음해요ㅠㅠ 
대략적으로는 이런 상황입니다~! 
![image](https://user-images.githubusercontent.com/65105537/130763627-6af3da54-2ea0-4610-a655-e36f3fced22e.png)

<UI 스크린샷> 
![image](https://user-images.githubusercontent.com/65105537/130762568-3a0acee8-ffc8-4a39-a5fc-906d7a6f645c.png)

**Reply by ha3158987** [reply]
> MyPage에 적용할 tab을 Material UI에서 가져왔는데 children을 어떻게 받아와서 쓰는지 파악하려고 고전중인 상태입니다...ㅠㅠ
> 문법 때문인건지 뭔가 하나에서 너무 헤매는 게 아닌가 싶어서 회의때 같이 코드를 봐주셨음해요ㅠㅠ
> 대략적으로는 이런 상황입니다~!
> ![image](https://user-images.githubusercontent.com/65105537/130763627-6af3da54-2ea0-4610-a655-e36f3fced22e.png)
> 
> <UI 스크린샷>
> ![image](https://user-images.githubusercontent.com/65105537/130762568-3a0acee8-ffc8-4a39-a5fc-906d7a6f645c.png)

해결!!!👍
